src/mugs.js

```diff
@@ -573,7 +574,7 @@ define([
             }
 
             if (!itextItem || lang === '_ids') {
-                return this.getNodeID();
+                return '[' + nodeID + ']';
```

Comment:
Sounds fine to me - I agree that it's getting to be a lot of brackets.

What do you think of the possibility of removing brackets altogether?